homeassistant/components/growatt_server/sensor/total.py

```diff
@@ -26,13 +26,15 @@
         api_key="todayEnergy",
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
Please check https://developers.home-assistant.io/docs/core/entity/sensor?_highlight=state_class#entities-representing-a-total-amount and explain why you chose to use `TOTAL` over `TOTAL_INCREASING` and/or `last_reset`

Reply:
I chose `Total` since energy won't change during the day, but if I missed this fact I can change it to `TOTAL_INCREASING`

Reply:
to me that sounds like it should be `TOTAL` with `last_reset` every time it resets at night